ykrt/src/frame/llvmbridge.rs

```diff
@@ -2,6 +2,8 @@ use llvm_sys::core::*;
 use llvm_sys::prelude::{LLVMBasicBlockRef, LLVMModuleRef, LLVMTypeRef, LLVMValueRef};
 use llvm_sys::target::{LLVMGetModuleDataLayout, LLVMTargetDataRef};
 use llvm_sys::LLVMTypeKind;
+use std::ffi::CStr;
```

Comment:
Prefer `use std::{ffi::CStr, fmt}`

Reply:
Fixed in [6e8780d](https://github.com/ykjit/yk/pull/769/commits/6e8780dfe8977b43fcb04c0d4586ced4dd00c93f)